exchanges/kucoin/kucoin_websocket.go

```diff
@@ -668,6 +663,10 @@ func (ku *Kucoin) processOrderChangeEvent(respData []byte) error {
 		LastUpdated:     response.Timestamp.Time(),
 		Pair:            pair,
 	}
+	ku.Websocket.DataHandler <- &orderChange
+	marginOrderChange := orderChange
```

Comment:
This will need to dereference the pointer because the next line will change the asset type and the websocket routine manager could handle it at a different time. 

And below